target/linux/ath79/dts/qca9563_tplink_cpe710-v1.dts

```diff
@@ -0,0 +1,142 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "qca956x.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	model = "TP-Link CPE710 v1";
+	compatible = "tplink,cpe710-v1", "qca,qca9563";
+
+	aliases {
+		label-mac-device = &eth0;
+		led-boot = &led_lan;
+		led-failsafe = &led_lan;
+		led-upgrade = &led_lan;
+		};
+
+	leds {
+		compatible = "gpio-leds";
+
+		led_lan: lan {
+			label = "blue:lan";
+			gpios = <&gpio 5 GPIO_ACTIVE_LOW>;
+		};
+
+		wlan5g {
+			label = "blue:wlan5g";
+			gpios = <&gpio 1 GPIO_ACTIVE_LOW>;
+			linux,default-trigger = "phy0tpt";
+		};
+	};
+
+	keys {
+		compatible = "gpio-keys";
+
+		reset {
+			label = "Reset button";
```

Comment:
label property should not have a space, but I would just delete the line

according to docs, if there is no label property, it takes the node name as the label (in this case, "reset")
the only reason the label property is needed is so Openwrt scripts can systematically control the LEDs

Reply:
Some context on what all the other DTS files are doing here, `label = "reset"` seems to be the current winner :)

```bash
$ grep -rh -B1 KEY_RESTART target/linux/ath79/dts | grep label | sort | uniq -c
    101             label = "reset";
      7             label = "Reset";
     50             label = "Reset button";
```

Reply:
@jwmullally The CPE and WBS Devices that have it specified all use "Reset button"

Reply:
> according to docs, if there is no label property, it takes the node name as the label (in this case, "reset")
the only reason the label property is needed is so Openwrt scripts can systematically control the LEDs

@mpratt14 I thought this was only the case for LEDs? Can you point me to the docs in question?